packages/realtime-server/src/state/store.ts

```diff
@@ -0,0 +1,37 @@
+import { BehaviorSubject, Observable } from 'rxjs';
+
+import { Action } from './actions';
+import {
+  SessionsState,
+  initialSessionsState,
+  sessionsReducer,
+} from './sessions';
+
+type State = {
+  sessions: SessionsState;
+};
+
+const initialState = {
+  sessions: initialSessionsState,
+};
+
+const reducer = (state: State, action: Action, sessionId: string): State => {
+  return {
+    sessions: sessionsReducer(state.sessions, action, sessionId),
+  };
+};
+
+type Store = BehaviorSubject<State>;
+
+export const createStore = (): Store => new BehaviorSubject(initialState);
+
+export const createDispatch =
+  (sessionId: string, store: Store) => (action: Action) => {
+    store.next(reducer(store.getValue(), action, sessionId));
+  };
+
+export const createObserve =
```

Comment:
whats the purpose of this fn? if you have the store already to call this with you could set up whatever observable you wanted with it